spec/fixtures/product_tree.json

```diff
@@ -0,0 +1,68 @@
+{
+  "identifier": "SLES",
+  "version": 15,
+  "recommended": false,
+  "arch": "x86_64",
+  "isbase": true,
+  "extensions": [
+    {
+      "identifier": "Some extension 1",
+      "version": 98,
+      "recommended": false,
+      "arch": "x86_64",
+      "extensions": []
+    },
+    {
+      "identifier": "Recommended extension 1",
+      "version": 15,
+      "recommended": true,
+      "arch": "x86_64",
+      "extensions": [
+          {
+            "identifier": "Some extension 2",
+            "version": 83,
+            "arch": "x86_64",
+            "recommended": false,
+            "extensions": []
+          },
+          {
+            "identifier": "Recommended extension 2",
```

Comment:
It would be nice to have identifiers without any repeating numbers. I've dumped the tree and it's kinda hard to figure out what's what in this output:
```
["Some extension 1",
 "Recommended extension 1",
 "Some extension 2",
 "Recommended extension 2",
 "Recommended extension 3",
 "Some extension 3",
 "Some extension 4",
 "Some extension 5"]
```
Maybe something along the lines of `lvl1_extension_1`, `lvl2_extension2`, `lvl2_recommended_extension3`, `lvl1_extension4` and so on?